running.html

```diff
@@ -72,14 +71,19 @@ <h3 id="running-vs">Running in Visual Studio (for debugging)</h3>
       </ol>
 
 
-      <!-- No Linux client available, no point including this right now
-      <h3 id="running-linux">Running on Linux</h3>
+      <h3 id="running-mac">Running on macOS</h3>
 
-      <ol>
-        <li><p>Copy the folders <tt>avi</tt>, <tt>dat</tt>, and <tt>sfx</tt> from your existing MOULa installation to the MOUL-OS folder.</p></li>
+      <ol start="3">
+        <li><p>Open a terminal window in your MOUL-OS folder.</p></li>
 
-        <li><p>Get the appropriate <tt>server.ini</tt> file for the server you will be connecting to.</p></li>
+        <li><p>Run <code>./plClient.app/Contents/MacOS/plClient --LocalData</code> to start the client.</p></li>
```

Comment:
@colincornaby Is this the best option, or does something like `open plClient.app --LocalData` work?

Reply:
On my mini, I've been using `open -n ./plClient.app --args -LocalData`.

Reply:
I've been using `NoSelfPatch` so that I download all the game data without looking for a client patch.

(Curious if that's the intended behavior or if maybe there is a difference between Mac/Windows.)

Reply:
`--LocalData` will skip all patching and use local game data and python files.
`--NoSelfPatch` will (on macOS) skip client patching, but depends on the server having the data files for all the game content (which is not guaranteed in the case of things like local Dockersand shards)